core/src/test/java/org/mobilitydata/gtfsvalidator/input/GtfsInputTest.java

```diff
@@ -66,4 +69,24 @@ public void zipInput() throws IOException {
         GtfsInput gtfsInput = GtfsInput.createFromPath(zipFile.getAbsolutePath());
         assertThat(gtfsInput.getFilenames()).containsExactly("stops.txt");
     }
+
+    @Test
+    public void createFromValidUrlShouldNotThrowException() throws IOException, URISyntaxException, InterruptedException {
+        GtfsInput underTest = GtfsInput.createFromUrl(
+                new URL("https://octa.net/current/google_transit.zip"),
```

Comment:
Sure. Why though ❔

Reply:
I picked this one: https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/usecase/src/test/resources/valid_zip_sample.zip

Reply:
>Sure. Why though ❔

Because OCTA could change their URL and break the test. If it's under our control it's less likely to be broken randomly.

>I picked this one: https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/usecase/src/test/resources/valid_zip_sample.zip

👍  Note you need to use the "raw" version of the URL though:
https://github.com/MobilityData/gtfs-validator/raw/v1.4.0/usecase/src/test/resources/valid_zip_sample.zip

Reply:
You can get the "raw" version by right-clicking on "Download" button in the GitHub UI and copying the link.

Reply:
Thanks!